src/locale/locales/fr/messages.po

```diff
@@ -2199,7 +2192,7 @@ msgstr "Affine les fils de discussion."
 
 #: src/screens/StarterPack/Wizard/index.tsx:202
 msgid "Finish"
-msgstr ""
+msgstr "Finir"
```

Comment:
We nearly always translate `Finish` (the verb) with `Terminer` in UI texts in French.

```suggestion
msgstr "Terminer"
```